testng-core/src/test/kotlin/org/testng/internal/ClassHelperTest.kt

```diff
@@ -0,0 +1,153 @@
+package org.testng.internal
+
+import org.assertj.core.api.Assertions.assertThat
+import org.testng.Reporter
+
+import java.net.URL
+import java.net.URLClassLoader
+import java.util.UUID
+import org.testng.annotations.DataProvider
+import org.testng.annotations.Test
+import org.testng.internal.samples.classhelper.issue1339.BabyPandaSample
+import org.testng.internal.samples.classhelper.issue1339.LittlePandaSample
+import org.testng.internal.samples.classhelper.issue1456.TestClassSample
+import org.testng.internal.samples.classhelper.misamples.AbstractMovesSample
+import org.testng.internal.samples.classhelper.misamples.BatmanSample
+import org.testng.internal.samples.classhelper.misamples.JohnTravoltaMovesSample
+import org.testng.internal.samples.classhelper.misamples.MickJaggerSample
+import org.testng.xml.XmlClass
+import org.testng.xml.XmlSuite
+import org.testng.xml.XmlTest
+import kotlin.reflect.KClass
+
+class ClassHelperTest {
+
+    @Test(dataProvider = "testDataProvider")
+    fun runFindClassesInSameTestScenarios(
+        scenarioName: String,//This parameter will help us differentiate scenarios in reports
+        classToBeFound: KClass<*>,
+        expectedCount: Int, vararg classes: KClass<*>
```

Comment:
fyi, trailing comma is possible with Kotlin and will improve the futur diff in case of change.

Reply:
Fixed